### ISSUE TITLE
RemoteConfig fails if remote config file requires custom cops

### ISSUE THREAD

**Issue by tiev** [rubocop-hq/rubocop#6707]
I'm trying to use a shared remote config file for multiple projects.

The remote config file requires some custom cops.

--------

## Expected behavior

Running rubocop will download the remote custom cops in the remote config file.

## Actual behavior

Rubocop tries to require files in local.

## Steps to reproduce the problem

In remote `.rubocop.yml`

```yml
inherit_from:
  - ./cops/.config.yml

require:
  - ./cops/my_cop/first_cop.rb
```

In remote `./cops/.config.yml`

```yml
MyCop/FirstCop:
  Enabled: true
```

In remote `./cops/my_cop/first_cop.rb`

```ruby
module MyCop
  class FirstCop < ::RuboCop::Cop::Cop
    def on_send(node)
      # ....
    end
  end
end
```

In local `.rubocop.yml`

```yml
inherit_from:
  - url_to_remote_config_file
```

## RuboCop version

```
$ rubocop -V
0.63.1 (using Parser 2.6.0.0, running on ruby 2.3.3 x86_64-darwin18)
```

**Comment by pocke** [comment]
I think we should be careful to introduce this feature. If we implement this feature, our config loader will be more complicated.
We can use custom cop gem instead.
https://rubocop.readthedocs.io/en/latest/extensions/
Or maybe we can also use inherit_gem instead, but I haven't confirmed it.
https://rubocop.readthedocs.io/en/latest/configuration/#inheriting-configuration-from-a-dependency-gem

I think creating a gem is the right way for this case, so personally I think requiring remote file is unnecessary. But I'd like to hear other core team member's opinion.

**Comment by tiev** [comment]
Yes, this is to report the case. I can achieve custom cops by using custom cop gems.

**Comment by Drenmi** [comment]
Thanks for the report @tiev. 🙇 I agree with @pocke here. We do want better support for custom cops that are single project only, but for cops used across multiple projects, a gem would be the way to go for now.
